Update solution.sql

markovsz opened the pull request:
Sql tesk task

markovsz committed:
Update solution.sql

AnnaGolosova commented on solution.sql:
```diff
@@ -1 +1,319 @@
+create database BankDb;
 
+create table Banks (
+    Id int not null,
```
Find a way to not populate Primary Key value manually with every INSERT operation

AnnaGolosova commented on solution.sql:
```diff
@@ -1 +1,319 @@
+    Id int not null,
+    BankId int not null,
+    CityId int not null,
+    StreetId int not null,
```
There's no need to move the street to a separate table. Just add "Address"

AnnaGolosova commented on solution.sql:
```diff
@@ -1 +1,319 @@
+    FirstName nvarchar(30) not null,
+    LastName nvarchar(30) not null,
+    MidName nvarchar(30),
+    SocialStatusId int not null,
```
I guess you missed Foreign Key here

AnnaGolosova commented on solution.sql:
```diff
@@ -1 +1,319 @@
+
+/*Задания*/
+
+/*Задание №1*/
```
Please, add here task requirenments

markovsz committed:
Update solution.sql

task comments were added, pk identity were added, street and buldingNumber were replaced to address
